packages/app/src/navigation/SidebarNavigation.vue

```diff
@@ -0,0 +1,37 @@
+<!-- This example requires Tailwind CSS v2.0+ -->
+<template>
+  <div class="flex-1 flex flex-col min-h-0 bg-gray-800">
+    <div class="flex-1 flex flex-col pt-5 pb-4 overflow-y-auto">
+      <div class="flex items-center flex-shrink-0 px-4">
+        <i-bi-bookmark-star class="text-white w-18px h-18px"/>
+      </div>
+      <nav class="mt-5 flex-1 px-2 bg-gray-800 space-y-1" aria-label="Sidebar">
+        <router-link custom v-slot="{ href, isActive }" v-for="item in navigation" :key="item.name" :to="item.href">
```

Comment:
Should this be `RouterLink`? We should either use PascalCase or kebab-case, but not a mix.

I wonder if we can lint for this.

Reply:
Whoops it blended in with `nav`. TitleCase is correct.

Reply:
Well, hmm, I don't know... our Icons will have to be kebab-case. I think maybe:

* Native HTML elements and built-in Vue components can be in kebab-case
* Icon components are okay (`<i-cy-my-icon-name />`)

Reply:
Sure, so in this case `RouterLink` should be title case?